regolith/helpers/a_expensehelper.py

```diff
@@ -25,34 +25,33 @@ def subparser(subpi):
                        default=None
                        )
     subpi.add_argument("-b", "--business",
-                       help="expense type is business"
-                       )
-    subpi.add_argument("-t", "--travel",
-                       help="expense type is travel"
+                       help="expense type is business. If not specified defaults to travel"
                        )
     subpi.add_argument("-y", "--payee",
                        help="payee of the expense, defaults to sbillinge"
                        )
     subpi.add_argument("-g", "--grants", nargs="+",
-                       help="grant or (occasionally) list of grants that cover this expense"
+                       help="list of grants that cover this expense"
```

Comment:
sorry, I may have mistyped...."grant, or list of grants, that cover this expense"   Let's have  default of "tbd"

Reply:
done